Lesson14/Akan/script.js

```diff
@@ -0,0 +1,192 @@
+
+  // 9. Convert the number 100 to a string and log the result.
+  // CODE HERE
+  const number = 100;
+  const numberString = number.toString();
+  console.log(number, numberString);
+  // This will create a string from a number and a string!!! console.log(number + numberString);
+
+  // 10. Convert the string "50" to a number and log its type to confirm the conversion.
+  // CODE HERE
+  const string = "50";
+
+  console.log(parseInt(string));
+
+  // 11. Generate a random integer between 0 and 10 and log it.
+  // CODE HERE
+  const randomNumber = Math.random() * 11; // 0 -> 10.99999
+  console.log('Random number ex. 11', Math.floor(randomNumber));
+
+  // 12. Round the number 3.7 down using Math.floor and 3.2 up using Math.ceil, log both.
+  // CODE HERE
+  const number1Ex12 = Math.floor(3.7);
+  console.log(number1Ex12);
+  const number2Ex12 = Math.ceil(3.2);
+  console.log(number2Ex12);
+
+
+  // Prompt can only be used in browser environment
+  const userResponse = prompt("What is your age?");
+  const userResposeNumber = parseInt(userResponse);
+
+  if(isNaN(userResposeNumber)) {
+    throw new Error("Please enter a valid number");
+} else {
+  console.log('userResponse', Math.floor(userResposeNumber));
+}
+
+// 13. Declare a boolean variable isStudent = true. Log it.
+// CODE HERE
+const isStudent = true;
+console.log(isStudent);
+
+
+// 14. Initialize counter = 0, then increment it by 1 using counter++ and log it.
+// CODE HERE
+let counter = 0;
+counter++;
+console.log(counter);
+
+// 15. Initialize points = 10, add 5 to it using points += 5, then log points.
+// CODE HERE
+let points = 10;
+points += 5;
+console.log(points);
+
+// 16. Declare name="Alice", age=30, city="Paris". Log "Alice (30) lives in Paris" using template literals.
+// CODE HERE
+const name = "Alice";
+const age = 30;
+const city = "Paris";
+console.log('${name}', '(${age}', 'lives in ${city}';
+
+
+// 17. Declare variables x=5, y=10, z=15. Log their total sum.
+// CODE HERE
+const x = 5;
+const y = 10;
+const z = 15;
+console.log(x + y + Z);
+
+// 18. Declare dividend=10 and divisor=3. Log the quotient (divisionResult) and difference (differenceResult).
+// CODE HERE
+const dividend = 10;
+const divisor = 3;
+const divisionResult = dividend / divisor;
+console.log(divisionResult);
+
+// 19. Declare firstName and lastName. Create fullName by concatenating them with a space and log it.
+// CODE HERE
+const firstName = "Mila";
+const lastName = "Degirmenci";
+const fullName = firstName + " " + lastName;
+console.log(fullName);
+
+// 20. Declare firstFactor=7 and secondFactor=2. Log the product.
+// CODE HERE
+const firstFactor = 7;
+cosnt secondFactor = 2;
+console.log(firstFactor * secondFactor);
+
+
+// 21. Log the value of Math.PI.
+// CODE HERE
+console.log(Math.PI);
+
+
+// 22. Declare counter=0. Increment it using three different methods (e.g., counter++, counter+=1, counter=counter+1) and log the result each time.
+// CODE HERE
+const counter = 0;
```

Comment:
`counter` should be declared as let instead of const since its value is being reassigned.

```suggestion
let counter = 0;
```